scipy/sparse/linalg/_special_sparse_arrays.py

```diff
@@ -19,37 +19,37 @@ class LaplacianNd(LinearOperator):
         A tuple of integers of length `N` (corresponding to the dimension of
         the Lapacian), where each entry gives the size of that dimension. The
         Laplacian matrix is square of the size ``np.prod(grid_shape)``.
-    boundary_conditions: string
+    boundary_conditions: {'neumann', 'dirichlet', 'periodic'}, optional
```

Comment:
```suggestion
    boundary_conditions : {'neumann', 'dirichlet', 'periodic'}, optional
```

The space before the colon is important.